test/python/pulse/test_block.py

```diff
@@ -27,7 +28,7 @@ class BaseTestBlock(QiskitTestCase):
     def setUp(self):
         super().setUp()
 
-        self.backend = FakeOpenPulse2Q()
+        self.backend = FakeArmonk()
```

Comment:
I had seen the test file and there was no reference of `self.backend` other than assigning it to the test class. Since the test only required whether the pulse schedule block was working or not, I tried to use the most basic pulse enabled backend (1-qubit armonk) for that. The tests were passing so I went along with that choice.

Reply:
I would avoid changing things in a test file unrelated to what you're testing. If you want to test with armonk in your test (which is a good idea), I would just do `backend = FakeArmonk()` in the test method and not change the backend used by the other tests.

Reply:
Okay, I shall update that. Thank you!